packages/experimental/src/components/Tearsheet/Tearsheet.js

```diff
@@ -0,0 +1,170 @@
+//
+// Copyright IBM Corp. 2020, 2020
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import cx from 'classnames';
+
+import { expPrefix } from '../../global/js/settings';
+
+import { ModalHeader, ModalBody } from 'carbon-components-react';
+
+import { TearsheetShell } from './TearsheetShell';
+
+const blockClass = `${expPrefix}-tearsheet`;
+
+export const Tearsheet = ({
+  buttons,
+  children,
+  className,
+  closeIconDescription,
+  description,
+  hasCloseIcon,
+  height,
+  influencer,
+  influencerPosition,
+  influencerWidth,
+  label,
+  navigation,
+  onClose,
+  open,
+  preventCloseOnClickOutside,
+  title,
+}) => {
+  const closeClasses = cx({
+    [`${blockClass}--header--no-close-icon`]: !hasCloseIcon,
+  });
+
+  const influencerClasses = cx({
+    [`${blockClass}--influencer`]: true,
+    [`${blockClass}--influencer--right`]: influencerPosition === 'right',
+    [`${blockClass}--influencer--wide`]: influencerWidth === 'wide',
+  });
+
+  return (
+    <TearsheetShell
+      className={className}
+      height={height}
+      onClose={onClose}
+      open={open}
+      preventCloseOnClickOutside={preventCloseOnClickOutside}
+      size="wide">
+      <ModalHeader
+        className={`${blockClass}--header`}
+        closeClassName={closeClasses}
+        iconDescription={closeIconDescription}
+        label={label}
+        title={title}>
+        {description && (
+          <div className={`${blockClass}--header-description`}>
+            {description}
+          </div>
+        )}
+        {navigation && (
+          <div className={`${blockClass}--header-navigation`}>{navigation}</div>
+        )}
+      </ModalHeader>
+      <ModalBody className={`${blockClass}--body`}>
+        {influencer && <div className={influencerClasses}>{influencer}</div>}
+        <div className={`${blockClass}--right`}>
+          {children && <div className={`${blockClass}--main`}>{children}</div>}
+          {buttons && <div className={`${blockClass}--buttons`}>{buttons}</div>}
+        </div>
+      </ModalBody>
+    </TearsheetShell>
+  );
+};
+
+Tearsheet.propTypes = {
+  /**
+   * Specifies the content for the buttons section of the Tearsheet.
+   * Optional.
+   */
+  buttons: PropTypes.node,
+  /**
+   * Specifies the content of the Tearsheet body. Optional.
+   */
+  children: PropTypes.node,
+  /**
+   * Specifies class(es) to be applied to the top-level Tearsheet node.
+   * Optional.
+   */
+  className: PropTypes.string,
+  /**
+   * The description for the close icon. Optional.
+   */
+  closeIconDescription: PropTypes.string,
+  /**
+   * Specifies the description of the Tearsheet.
+   * Optional.
+   */
+  description: PropTypes.node,
```

Comment:
Why is this not PropTypes.string?

Reply:
So, that rich content can be provided if needed. It is standard practice for the ModalHeader etc to allow nodes for label, title etc, so is consistent with that.